index.js

```diff
@@ -38,23 +38,35 @@ var PROGRAMS = {
 function runVersionCommand(command, callback) {
   exec(command, function(execError, stdin, stderr) {
     var commandDescription = JSON.stringify(command);
-    if (execError && execError.code === 127) {
-      return callback(null, {
-        notfound: true,
-      });
-    }
-    else if (execError) {
-      var runError = new Error("Command failed: " + commandDescription);
-      if (stderr) {
-        runError.stderr = stderr.trim();
+
+    if (execError) {
+      if ((process.platform === 'win32') && (execError.code === 1)) {
+        if (stderr.indexOf('is not recognized')) {
+          execError.code = 127;
```

Comment:
Minor suggestion, but I'd prefer to return early here with a `notfound` callback call rather than mutating `execError` with a fake error code.

Reply:
Done 👍